src/main/java/techcourse/myblog/presentation/ArticleController.java

```diff
@@ -0,0 +1,55 @@
+package techcourse.myblog.presentation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import techcourse.myblog.domain.Article;
+import techcourse.myblog.service.ArticleService;
+
+import javax.transaction.Transactional;
+
+@Controller
+@RequestMapping("/articles")
+public class ArticleController {
+    private final ArticleService articleService;
+
+    @Autowired
```

Comment:
단일 생성자인 경우, `@Autowired`가 없어도 됩니다.

> If a class only declares a single constructor to begin with, it will always be used, even if not annotated.

```suggestion

```